benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java

```diff
@@ -0,0 +1,522 @@
+  private ByteBuf genericRequest;
+
+  private final Control.ClientConfig config;
+  private final AbstractIntegerDistribution distribution;
+  private volatile boolean shutdown;
+  private final int threadCount;
+
+  ManagedChannel[] channels;
+  BenchmarkServiceGrpc.BenchmarkServiceBlockingStub[] blockingStubs;
+  BenchmarkServiceGrpc.BenchmarkServiceStub[] asyncStubs;
+  Recorder recorder;
+  private ExecutorService fixedThreadPool;
+  private Messages.SimpleRequest simpleRequest;
+  private final OperatingSystemMXBean osBean;
+  private long lastMarkCpuTime;
+
+  LoadClient(Control.ClientConfig config) throws Exception {
+    LOG.log(Level.INFO, "Client Config \n" + config.toString());
+    this.config = config;
+    // Create the channels
+    channels = new ManagedChannelImpl[config.getClientChannels()];
+    for (int i = 0; i < config.getClientChannels(); i++) {
+      channels[i] =
+          Utils.newClientChannel(
+              Epoll.isAvailable() ?  Transport.NETTY_EPOLL : Transport.NETTY_NIO,
+              Utils.parseSocketAddress(config.getServerTargets(i % config.getServerTargetsCount())),
+              config.hasSecurityParams(),
+              config.hasSecurityParams() && config.getSecurityParams().getUseTestCa(),
+              config.hasSecurityParams()
+                  ? config.getSecurityParams().getServerHostOverride() :
+                  null,
+              true,
+              Utils.DEFAULT_FLOW_CONTROL_WINDOW,
+              false);
+    }
+
+    // Create a stub per channel
+    if (config.getClientType() == Control.ClientType.ASYNC_CLIENT) {
+      asyncStubs = new BenchmarkServiceGrpc.BenchmarkServiceStub[channels.length];
+      for (int i = 0; i < channels.length; i++) {
+        asyncStubs[i] = BenchmarkServiceGrpc.newStub(channels[i]);
+      }
+    } else {
+      blockingStubs = new BenchmarkServiceGrpc.BenchmarkServiceBlockingStub[channels.length];
+      for (int i = 0; i < channels.length; i++) {
+        blockingStubs[i] = BenchmarkServiceGrpc.newBlockingStub(channels[i]);
+      }
+    }
+
+    // Determine no of threads
+    if (config.getClientType() == Control.ClientType.SYNC_CLIENT) {
+      threadCount = config.getOutstandingRpcsPerChannel() * config.getClientChannels();
+    } else {
+      threadCount = config.getAsyncClientThreads() == 0
+          ? Runtime.getRuntime().availableProcessors()
+          : config.getAsyncClientThreads();
+    }
+    // Use a fixed sized pool of daemon threads.
+    fixedThreadPool = Executors.newFixedThreadPool(threadCount,
+        new DefaultThreadFactory("client-worker", true));
+
+    // Create the load distribution
+    if (config.getLoadParams().getClosedLoop() != null) {
+      distribution = new EnumeratedIntegerDistribution(new int[]{0}, new double[]{1});
+    } else if (config.getLoadParams().getPoisson() != null) {
+      distribution =  new PoissonDistribution(config.getLoadParams().getPoisson().getOfferedLoad()
```

Comment:
In that case you don't really need to pull in the math dependency, as there is an easy way to create an exponentially distributed random variable, from a uniformly distributed random variable, as we did in the `OpenLoopClient`: https://github.com/grpc/grpc-java/blob/master/benchmarks/src/main/java/io/grpc/benchmarks/qps/OpenLoopClient.java#L232